docs: some steps to use gh pages (#1059)
flame-engine/flame: doc/platforms.md

$ flutter build web --release --web-renderer canvaskit
```

## Deploy your game to GitHub Pages

> One easy way to deploy your game online, is to use [GitHub Pages])https://pages.github.com/).

Reviewer: ```suggestion
One easy way to deploy your game online, is to use [GitHub Pages](https://pages.github.com/).
```

> One easy easy way to deploy your game online, it to use [GitHub Pages])https://pages.github.com/).

Reviewer 2: ```suggestion
One easy way to deploy your game online, is to use [GitHub Pages])https://pages.github.com/).
```

> It is a cool feature from GitHub, where you can easily host web content from your repository.

Reviewer: > It is a cool feature from GitHub

You startled the ~witch~ @wolfenrain

Reviewer: *by which

Author: ```suggestion
It is a cool feature from GitHub, by which you can easily host web content from your repository.
```

> Here we will explain the easiest way to get your game hosted using that.

Reviewer 2: ```suggestion
Here we will explain the easiest way to get your game hosted using GitHub pages.
```

First thing, lets create the branch where your deployed files will live:

```bash
git checkout -b gh-pages
```

> This branch be created from `main` or any other place, it doesn't matter much. After you push that

Author: ```suggestion
This branch can be created from `main` or any other place, it doesn't matter much. After you push that
```

branch go back to your `main` branch.

Now you should add the [flutter-gh-pages](https://github.com/bluefireteam/flutter-gh-pages)
action to your repository, you can do that by creating a file `gh-pages.yaml` under the folder
`.github/workflows`.

```yaml
name: Gh-Pages

on:
  push:
    branches: [ main ]

jobs:
  build:
    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v2
      - uses: subosito/flutter-action@v1
      - uses: bluefireteam/flutter-gh-pages@v7
        with:
          baseHref: /NAME_OF_YOUR_REPOSITORY/
          webRenderer: canvaskit
```

Be sure to change `NAME_OF_YOUR_REPOSITORY` to the name of your GitHub repository.

> Now, whenever you commit something on the `main` branch, the action will run and updated your

Reviewer 2: ```suggestion
Now, whenever you push something to the `main` branch, the action will run and update your
```
